packages/cashier/src/components/cashier-search-box/cashier-search-box.jsx

```diff
@@ -0,0 +1,73 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Field as FormField, Formik, Form } from 'formik';
+import { Icon, Input } from '@deriv/components';
+import './cashier-search-box.scss';
+
+const CashierSearchBox = ({ className, onClear, onSearch, placeholder, setIsSearchLoading }) => {
+    React.useEffect(() => {
+        return onClear;
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
```suggestion
    }, [onClear]);
```

Reply:
@farzin-fs when we add this dependency the page is hangup because of constant rerenders, that's why i remove onClear from dependency array